src/test/java/edu/mit/dos/object/ObjectServiceIT.java

```diff
@@ -40,7 +51,8 @@ public void testPost() {
         map.add("title", "Tomb of Sitta Zubayda");
         HttpEntity<MultiValueMap<String, String>> request = new HttpEntity<>(map, new HttpHeaders());
         String body = this.restTemplate.postForObject("/object",  request,String.class);
-        assertThat(body).isEqualTo("ok");
+        assertThat(body).isNotNull();
```

Comment:
This should test for an oid (if that's what being returned). You could do a get call following this to confirm that the object has the title for example.

Reply:
But we won't know what oid will be though since it will be generated when the post happens, right? That's why I wasn't sure what to test for other than it returns a string on line 55

Reply:
So the response of this.restTemplate.postForObject should be an oid. Using this oid you can make a call to the get endpoint, retrieve the object, and confirm that it's title, etc., matches the one POSTed.